src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,73 @@
+import classNames from 'classnames';
+import React from 'react';
+import { FilterType } from '../../types/FilterType';
+
+type Props = {
+  filterType: FilterType,
+  setFilterType: (filter: FilterType) => void,
+  activeTodosQuantity: number,
+  completedTodosQuantity: number,
+  onClearCompleted: () => void
+};
+
+export const TodoFooter: React.FC<Props> = ({
+  filterType,
+  setFilterType,
+  activeTodosQuantity,
+  completedTodosQuantity,
+  onClearCompleted,
+}) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${activeTodosQuantity} items left`}
+      </span>
+
+      <nav className="filter">
+        <a
+          href="#/"
+          className={classNames(
+            'filter__link',
+            { selected: filterType === FilterType.All },
+          )}
+          onClick={() => setFilterType(FilterType.All)}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
```

Comment:
Instead of using 'href' with '#/active', consider using a proper navigation library like react-router or handle navigation programmatically.